docs/ui/auth/fragments/ionic/authenticator.md

```diff
@@ -50,6 +50,71 @@ Amplify UI Components use `slots` based off of the [Web Components slot element]
 | `totp-setup`           | Content placed inside of the totp-setup workflow for when a user opts to use TOTP MFA                                  |
 | `greetings`            | Content placed inside of the greetings navigation for when a user is signed in                                         |
 
+### Custom Form Fields
+
+If you'd like to customize the form fields in the Authenticator Sign In or Sign Up component you can do so by using the `formFields` property.
+
+The following is an example highlights the use of Authenticator with customized Sign Up form fields and [authentication with email](#authenticate-with-email-or-phone-number):
+
+*app.component.ts*
+```js
+import { Component } from '@angular/core';
+import { FormFieldTypes } from '@aws-amplify/ui-components';
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css'],
+})
+export class AppComponent {
+  formFields: FormFieldTypes;
+  constructor() {
+    this.formFields = [
+      {
+        type: "email",
+        label: "Custom email Label",
+        placeholder: "custom email placeholder",
+        required: true,
+      },
+      {
+        type: "password",
+        label: "Custom Password Label",
+        placeholder: "custom password placeholder",
+        required: true,
+      },
+      {
+        type: "phone_number",
+        label: "Custom Phone Label",
+        placeholder: "custom Phone placeholder",
+        required: false,
+      },
+    ];
+  }
+}
+```
+
+*app.component.html*
+```html
+<amplify-authenticator usernameAlias="email">
```

Comment:
we arent using the previous flag right? where it was `framework="Ionic"`?

Reply:
Correct.